.github/release.yml

```
@@ -0,0 +1,31 @@
+    - title: Documentation 📝
+      labels:
+        - documentation
+    - title: CI/CD & Infrastructure 🔧
+      labels:
+        - ci/cd
+    - title: Dependencies 📦
+      labels:
+        - dependencies
+    - title: Other Changes
+      labels:
+        - "*"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)